app/src/main/java/fr/free/nrw/commons/wikidata/WikidataDisambiguationItems.kt

```diff
@@ -0,0 +1,13 @@
+package fr.free.nrw.commons.wikidata
+
+
+enum class WikidataDisambiguationItems(val id: String) {
+    DISAMBIGUATION_PAGE("Q4167410"), INTERNAL_ITEM("Q17442446"), CATEGORY("Q4167836");
```

Comment:
Should we add comments stating why we choose these constants?

Reply:
I could add a link to [this comment](https://github.com/commons-app/apps-android-commons/issues/3077#issuecomment-625601555)? I don't think it is much deeper than "these are disambiguation items" which is what the classname says

Reply:
I think adding this link is good enough indeed.

Reply:
Yes that should be fine